test/vector_tests.erl

```diff
@@ -0,0 +1,96 @@
+%% -------------------------------------------------------------------
```

Comment:
I'm assuming this file is left untouched except for the occasional static analysis -related fix. Is my assumption correct?

Reply:
Presumably, yes.